Attribution.txt

```diff
@@ -223,3 +223,15 @@ https://github.com/golang/tools/blob/master/LICENSE
 
 github.com/go-jose/go-jose/v3 (Apache-2.0) https://github.com/go-jose/go-jose
 https://github.com/go-jose/go-jose/blob/v3/LICENSE
+
+github.com/gabriel-vasile/mimetype (MIT) https://github.com/gabriel-vasile/mimetype
+https://github.com/gabriel-vasile/mimetype/blob/master/LICENSE
+
+github.com/klauspost/compress (Apache-2.0) https://github.com/klauspost/compress
+https://github.com/klauspost/compress/blob/master/LICENSE
+
+github.com/stretchr/objx (MIT) https://github.com/stretchr/objx
+https://github.com/stretchr/objx/blob/master/LICENSE
+
+google.golang.org/genproto/googleapis/rpc https://github.com/googleapis/go-genproto/blob/main/LICENSE
+https://github.com/googleapis/go-genproto/tree/main/googleapis/rpc
```

Comment:
Where are these coming from? You didn't change the go.mod explicit section.

Reply:
`make test` was still showing that I needed to include these for the attribution

Reply:
Yes, this is because the go-mod indirect section changed when nothing changed in the explicit section